cms/pages/templates/pages/base_page.html

```diff
@@ -4,36 +4,58 @@
 {% block body_class %}template-basepage{% endblock body_class %}
 
 {% block content %}
-
-<div class="nhsuk-width-container">
-    <div class="nhsuk-grid-row">
-        <div class="nhsuk-grid-column-full">
-            <h1>{{ self.title }}</h1>
-        </div>
-    </div>
-
-    <div class="nhsuk-grid-row">
-        <div class="nhsuk-grid-column-two-thirds">
-            {% for block in self.body %}
-                {% include_block block %}
-            {% endfor %}
-        </div>
-        <div class="nhsuk-grid-column-one-third">
-            {% if children %}
-            <nav class="nhsuk-contents-list" role="navigation" aria-label="Pages in this section" aria-labbeledby="sideNav">
-                <h2 id="sideNav" class="nhsuk-heading-s">Pages in this section</h2>
-                <ol class="nhsuk-contents-list__list">
-                    {% for child in children %}
-                    <li class="nhsuk-contents-list__item">
-                        <a class="nhsuk-contents-list__link" href="{{ child.url }}">{{ child }}</a>
-                    </li>
-                    {% endfor %}
-
-                </ol>
-            </nav>
-            {% endif %}
-        </div>
-    </div>
-
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+  </div>
+</div>
+{% if children %}
+	<div class="nhsuk-grid-row">
+		<div class="nhsuk-grid-column-one-third nhsuk-u-margin-top-4 content_nav">
+			<nav>
+				<h3 class="parent_page_item">
+					<a href="{{ self.get_parent.url  }}">
+						<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
There's a part of me wondering whether it's worth having a `{% include "chevron.svg.html" %}` to improve readability / reuse, but I don't think it's essential.